react/components/ShippingOptionList.tsx

```diff
@@ -64,7 +64,7 @@ const ShippingOptionList: React.FC<Props> = ({
       </span>
 
       <div className="mt2 mb6 lh-copy">
-        <PlaceDetails display="compact" />
+        <PlaceDetails display="minimal" />
```

Comment:
this would cause some weirdness when selecting an address from the address list and only see the zipcode in the SLA selection list, right?

Reply:
Good point. We will use the minimal display when the user enters a new address, which is always the case in the first purchase scenario. In second purchase scenarios this needs to be adjusted, but I guess I'll take care of that in a following PR.

Reply:
Right. I changed it to only use the `minimal` display when the address is complete, which for now I'm going to assume is equivalent to "isValid === true". Currently this is only not true when the user is identified but not authenticated, but I was thinking of changing `AddressContext` and force it to return `isValid = true` in this case. wdyt?

Reply:
sounds good!